bedrock-node/main.yml

```diff
@@ -71,5 +71,5 @@ jobs:
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@v1
       with:
-        file: ./test/coverage.lcov
+        file: ./test/coverage/lcov.info
```

Comment:
Yes, this is good, I discovered that this file was the same was the other only after I started working on the web modules.